work-scheduler-data/src/main/java/com/github/gaalgergo/workscheduler/data/entities/VacationRequestEnum.java

```diff
@@ -0,0 +1,14 @@
+package com.github.gaalgergo.workscheduler.data.entities;
+
+import javax.persistence.Entity;
+
+@Entity
+public enum VacationRequestEnum {
```

Comment:
1. You are storing both vacation request **types** (ill, holiday...) and vacation request **states** (Accepted, Rejected...) in the same enumeration. Why?

2. You have this enumeration but you don't use it anywhere.